lib/scout_apm/instruments/ecto_telemetry.ex

```diff
@@ -0,0 +1,21 @@
+defmodule ScoutApm.Instruments.EctoTelemetry do
+  @doc """
+  Attaches an event handler for Ecto queries.
+
+  Takes a fully namespaced Ecto.Repo module as the only argument. Example:
+
+      ScoutApm.Instruments.EctoTelemetry.attach(MyApp.Repo)
+  """
+  def attach(repo_module) do
+    query_event = repo_module
+                  |> Module.split()
+                  |> Enum.map(& &1 |> Macro.underscore() |> String.to_atom())
+                  |> Kernel.++([:query])
+
+    Telemetry.attach("scout-ecto-query-handler", query_event, ScoutApm.Instruments.EctoTelemetry, :handle_event, nil)
```

Comment:
Does that `scout-ecto-query-handler` string mean anything? Can we instrument multiple repos with this?